examples : add README.md to tts example [no ci]

danbev opened the pull request.

danbev committed:
examples : add README.md to tts example [no ci]

ggerganov commented on examples/tts/README.md:
```diff
@@ -0,0 +1,71 @@
+## Text To Speach (TTS) example
```
For consistency with the other example readmes:

```suggestion
# llama.cpp/example/tts

```

danbev replied:
I'll update this shortly :+1:

ggerganov reviewed (approved):
It would be nice to have a quick-start section at the beginning with the shortcut:

```bash
llama-tts --tts-oute-default -p "Hello world" && aplay output.wav
```

And then get into the details for model conversion.

danbev committed:
squash! examples : add README.md to tts example [no ci]

Fix heading to be consistent with other examples, and add a quickstart
section to README.md.

danbev committed:
squash! examples : add README.md to tts example [no ci]

Fix spelling mistake.

danbev commented:
> It would be nice to have a quick-start section at the beginning with the shortcut:

I've added a [section](https://github.com/ggerganov/llama.cpp/pull/11155/commits/9091993a5ecc071147cb844f97aa625a878dfd00#diff-28369cd7f93cc6c24b2ba6e41fda78e1a24dca74d7f8096f136ee159f06f3d9cR6) at the top for this, let me know if this is what you had in mind. Thanks

ggerganov commented:
Yes, this great.
